tedge/src/cli/connect/command.rs

```diff
@@ -352,17 +362,37 @@ fn get_bridge_config_file_path(bridge_config: &BridgeConfig) -> Result<String, C
     ])?)
 }
 
-fn start_and_enable_tedge_mapper(user_manager: &UserManager) {
+fn start_and_enable_tedge_mapper_c8y(user_manager: &UserManager) {
+    let mut failed = false;
+
+    println!("Starting tedge-mapper service.\n");
+    if let Err(err) = TedgeMapperC8yService.restart(user_manager) {
+        println!("Failed to stop tedge-mapper service: {:?}", err);
+        failed = true;
+    }
+
+    println!("Persisting tedge-mapper on reboot.\n");
+    if let Err(err) = TedgeMapperC8yService.enable(user_manager) {
+        println!("Failed to enable tedge-mapper service: {:?}", err);
+        failed = true;
+    }
+
+    if !failed {
+        println!("tedge-mapper service successfully started and enabled!\n");
+    }
+}
+
+fn start_and_enable_tedge_mapper_az(user_manager: &UserManager) {
```

Comment:
you aren't missing a root guard here?

```
let _root_guard = user_manager.become_user(ROOT_USER);
```

as in `stop_and_disable_tedge_mapper_c8y`